content/projects/gsoc/students.adoc

```diff
@@ -50,31 +81,205 @@ We highly recommend to make some contributions to the project while you work on
+
+Once the application period is over, administrators and mentors make a decision on which proposal to accept
+based on the proposal submitted to the Google Summer of Code website.
+Only proposals submitted before the deadline to the Google Summer of Code website are considered.
+
+We understand students are anxious to know whether they are selected or not, but admins and mentors are
+bound to secrecy until Google announces the selection results.
+We will not discuss the selection with students until Google makes the announcement.
+
+== Congratulations, you have been selected... or not
+
+We thank all students who reach out to us during the application period.
+If you have not been selected link:https://google.github.io/gsocguides/student/being-turned-down[read this],
+there could be many reasons, and some are even outside of our control.
+Do not feel bad, we encourage you to stay with the community, and apply again next year.
+
+If you have been selected, the community bonding period starts within two days afterthe announcement.
+
+== Community Bonding
+
+As soon as the students are accepted, the community bounding period starts.
+During this period, students are not expected to be coding immediately.
+Instead they are expected to prepare to code.
+
+A successful community bonding usually leads to successful coding periods.
+It is our experience that poor community bonding leads to difficult coding periods.
+
+Use the community bonding to:
+
+. Formalize the communication channels with your mentors:
+  * Setup a gitter chat room for the project (linked to github).
+. Setup the weekly meeting schedule with your mentors:
+  * Two meetings per week is recommended,
+  * Announce your meeting schedule to:
+  ** The link:https://groups.google.com/forum/#!forum/jenkinsci-gsoc-all-public[Jenkins GSoC mailing list].
+  ** The gitter chat of your project.
+  ** Send a google calendar meeting invite to the mentors, CC the org admins.
+. Get introduced to the key stakeholders and contributors in the area of the project by you mentors:
+  * For example, an introduction to subject matter experts.
+. Continue to discuss and plan the project with the community and the mentors:
+  * Work on the design document of the project.
+  ** Work on clarifying objectives and expectations,
+  ** Study, refine and discuss the design and the project plan,
+  ** Top-level architecture document:
+  *** Create diagrams of operation,
+  *** Answer questions such as "How is the user going to use this?", "What configurations are needed?", etc.,
+  *** Some people find it useful to write a mini user guide or how-to guide, as if the project was already done. This usually helps define the project.,
+  * Create an implementation plan with milestones per coding period.
+  * At this point it may be appropriate to discuss the project on the _jenkinsci-dev@googlegroups.com_ mailing list (link:https://groups.google.com/forum/#!forum/jenkinsci-dev[archives]). Talk to the mentor about it.
```

Comment:
```suggestion
  * At this point it may be appropriate to discuss the project on the _jenkinsci-dev@googlegroups.com_ mailing list or on the SIG mailing list. Talk to your mentors about it.
```

Reply:
Got it.